### PR TITLE
document dev environment setup

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by russellb** [comment]
Thanks, this also fixes https://github.com/metalkube/cluster-api-provider-baremetal/issues/14

File: docs/dev/setup.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+The cluster-api requires two external tools for running the tests
+during development.
+
+## Install kustomize
+
+```bash
+eval $(go env)
+export GOPATH
+./tools/install_kustomize.sh
```
Comment (russellb) [inline docs/dev/setup.md]: did you forget to `git add` these new scripts?

Comment (mhrivnak) [reply docs/dev/setup.md]: They're already present and being used on travis.

Comment (russellb) [reply docs/dev/setup.md]: nevermind, i missed these going in earlier

**Comment by russellb** [comment]
Can you add a link to this in the README?

**Comment by dhellmann** [comment]
> Can you add a link to this in the README?

Done

**Pull request merged**